apps/sms/views/conversation/style/attachment.css

```diff
@@ -171,23 +180,33 @@
 }
 
 .img-placeholder {
-  background-position: left -10rem;
+  background-position: left -10rem; /* BiDi proof: CSS sprites */
 }
 
 .audio-placeholder {
-  background-position: left -20rem;
+  background-position: left -20rem; /* BiDi proof: CSS sprites */
 }
 
 .video-placeholder {
-  background-position: left -30rem;
+  background-position: left -30rem; /* BiDi proof: CSS sprites */
 }
 
 .vcard-placeholder {
-  background: url('images/attachment-vcard.png') no-repeat left top / 4rem;
+  background: url('images/attachment-vcard.png') no-repeat;
+  background-size: 4rem;
 }
+html[dir="ltr"] .vcard-placeholder { background-position: left top; }
+html[dir="rtl"] .vcard-placeholder { background-position: right top; }
 
 .incoming.received .vcard-placeholder {
-  background: url('images/attachment-vcard-incoming.png') no-repeat left top / 4rem;
+  background: url('images/attachment-vcard-incoming.png') no-repeat;
+  background-size: 4rem;
+}
+html[dir="ltr"] .incoming.received .vcard-placeholder {
+  background-position: left top;
+}
+html[dir="rtl"] .incoming.received .vcard-placeholder {
+  background-position: right top;
 }
```

Comment:
can't we just use `.incoming.received .vcard-placeholder { background-url: url(right url); }` and get rid of the other properties ?